app/pages/index.tsx

```diff
@@ -25,12 +26,20 @@ const getPagesQuery = graphql`
     session {
       sub
     }
+    openIntake {
+      openTimestamp
+      closeTimestamp
+    }
   }
 `;
 
 // eslint-disable-next-line @typescript-eslint/ban-types
 const Home = ({ preloadedQuery }: RelayProps<{}, pagesQuery>) => {
-  const { session } = usePreloadedQuery(getPagesQuery, preloadedQuery);
+  const { session, openIntake } = usePreloadedQuery(
+    getPagesQuery,
+    preloadedQuery
+  );
+  console.log(openIntake)
```

Comment:
this lil guy snuck in here :sweat_smile:

Reply:
My achilles heel! Pushed a commit to fix